stripe/_list_object.py

```diff
@@ -222,7 +251,7 @@ def previous_page(
         params_with_filters.update({"ending_before": first_id})
         params_with_filters.update(params)
 
-        result = self.list(
+        result = self.list(  # pyright: ignore[reportDeprecated]
```

Comment:
Will this fire DeprecatedWarning for users even though they use a non-deprecated method?

Reply:
Oh, good catch. Need to wrap like above.